HelpSource/Classes/Object.schelp

```diff
@@ -48,20 +48,34 @@ Create a new instance. The creation of new instances of any Object actually happ
 
 method::newCopyArgs
 
-Creates a new instance and copies the arguments to the instance variables in the order that the variables were defined. If the superclass's code::new:: method requires arguments, the first arguments passed to code::newCopyArgs:: will be passed on to that method, and the following arguments will be copied to this class's instance variables. Class variables are ignored.
+Creates a new instance and copies the arguments to the instance variables in the order that the variables were defined.
+
+Importantly, super class constructors and NOT called, but the first values will be copied.
```

Comment:
... superclass constructors are NOT ...

Reply:
This was never the case. Perhaps it's a bug?

Reply:
I was just referring to spelling of _superclass_ and _are_.

But yes, if there's a doubt about whether args are supposed to propagate to the superclass, would be good to confirm that (I suspect not, this would obscure behavior).

Reply:
.... Hahaha! I proofed this too 😂